docs/geedocs/5.3.3/answer/releaseNotes/relNotesGEE5_3_3.html

```diff
@@ -139,6 +139,11 @@ <h1>Release notes: Open GEE 5.3.3<a class="headerlink" href="#release-notes-open
 resulting in lower memory utilization and significantly improved
 performance.</td>
 </tr>
+<tr class="row-even"><td>1577</td>
+<td>Use defusedxml in server code</td>
```

Comment:
Did we update the build [doc ](https://github.com/google/earthenterprise/blob/master/earth_enterprise/BUILD.md)to list this added dependency ?

Reply:
The new dependency is bundled, so there's no changes to the build docs.